setup.cfg

```diff
@@ -45,6 +45,8 @@ ec2 =
     boto3
 gce =
     apache-libcloud
+linode =
+    linode-python; python_version<"3.0"
```

Comment:
what about py3?

Reply:
It's an abandoned library basically using deprecated API bindings. There is a push for a new `linode_v4` module using the new API bindings library and when that lands in Ansible we'll be bringing that here. That python dependency in py2/py3.